src/js/common/reducers/index.js

```diff
@@ -13,5 +15,8 @@ export default combineReducers({
   health: combineReducers({
     rx
   }),
+  disability: combineReducers({
```

Comment:
@jbalboni -- I think thats a valid point. This is a first phase of trying to get everything into the commonStore and decrease the amount of calls that we are making to the api and then optimize. I think we are moving to a world where are are going to try to have notifications and that will require data from many apps on the same page. I think what we are trying to do is implement and then optimize.